MesaModel/model/Pupil.py

```diff
@@ -469,20 +560,31 @@ def changeState(self):
                 self.model.model_state.learning_count -= 1
             return 1
 
-    def set_start_math(self):
+    # Calculate the end maths score based on the time spent by a pupil in
+    # a learning state and the pupil's starting maths score. The formula is:
+    #   scaled_start_maths = (2.7 ^ start_maths) ^ (1/7.6)
+    #   total_learnt = learning_counter + scaled_start_maths
+    #   end_maths = (7.6 * log(total_learnt)) + pupil_ability
+    # Learning counter is incremented every time the end maths score is calculated.
+    # That is, it is not the same as the number of steps that the pupil has spent learning.
+    # Peter and Chris should be consulted to verify/decide if this is a good formula.
+    # The way in which the start maths score is scaled seems a bit arbitrary.
```

Comment:
Just looked at the paper which explains where the numbers come from:

> Start Math scaled: As number of ticks the students learn indicate here their final score in Math, we have rescaled the Start Math score to represent minutes of learning:
>
>     Smath_scaled (s,c)= (e^(Smath(s,c)) )^(1/n)
>
> We use n in the exponent to fit the logarithmic function to map the ‘learning Minutes’ into ‘Score’ in a similar manner as the work of [22], who used the logarithmic function to map ‘Teacher feedback’ into ‘Score’. To fit the logarithmic function, we use the total number of minutes the students would possibly have in a school year, which equals to end-time = 8550. Since `log⁡ (8550^n) = 69`, we calculate n to be `~= 7.621204857`.